sdk/appconfiguration/azure-data-appconfiguration/src/samples/java/com/azure/data/appconfiguration/ReadOnlySampleAsync.java

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.data.appconfiguration;
+
+import com.azure.data.appconfiguration.models.ConfigurationSetting;
+
+/**
+ * Sample demonstrates how to set and clear read-only a configuration setting asynchronously.
+ */
+public class ReadOnlySampleAsync {
+    /**
+     * Runs the sample algorithm and demonstrates how to set and clear read-only a configuration setting asynchronously.
+     *
+     * @param args Unused. Arguments to the program.
+     */
+    public static void main(String[] args) {
+        // The connection string value can be obtained by going to your App Configuration instance in the Azure portal
+        // and navigating to "Access Keys" page under the "Settings" section.
+        String connectionString = "endpoint={endpoint_value};id={id_value};secret={secret_value}";
+
+        // Instantiate a client that will be used to call the service.
+        final ConfigurationAsyncClient client = new ConfigurationClientBuilder()
+            .connectionString(connectionString)
+            .buildAsyncClient();
+
+        // Name of the key to add to the configuration service.
+        final String key = "hello";
+        final String value = "world";
+
+        client.setConfigurationSetting(key, null, value).subscribe(
+            result -> {
+                final ConfigurationSetting setting = result;
+                System.out.println(String.format("[SetConfigurationSetting] Key: %s, Value: %s", setting.getKey(), setting.getValue()));
```

Comment:
It would be better to us printf rather than println if you're going to format the string.

Reply:
Same with any other instances where you follow a println with String.format(